python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -422,19 +442,25 @@ cdef class ParquetWriter:
     cdef cudf_io_types.statistics_freq stat_freq
     cdef cudf_io_types.compression_type comp_type
     cdef object index
+    cdef size_t row_group_size_bytes
+    cdef size_type row_group_size_rows
 
-    def __cinit__(self, object filepaths_or_buffers, object index=None,
-                  object compression=None, str statistics="ROWGROUP"):
+    def __cinit__(self, object filepath_or_buffer, object index=None,
+                  object compression=None, str statistics="ROWGROUP",
+                  int row_group_size_bytes=13369344,
```

Comment:
```suggestion
                  int row_group_size_bytes=134217728,
```

Reply:
Wow - I'm sorry for that. I think your excellent code-review skills are making me too comfortable :)

Reply:
No worries! Teamwork makes the dream work, as they say. 👍